Alignment/OfflineValidation/python/TkAlAllInOneTool/primaryVertexValidationTemplates.py

```diff
@@ -0,0 +1,374 @@
+                                                                         maxD0Significance = cms.double(5.0),                        # fake cut (requiring 1 PXB hit)
+                                                                         minPt = cms.double(0.0),                                    # better for softish events     
+                                                                         trackQuality = cms.string("any")
+                                                                         ),
+                                        
+                                           TkClusParameters = cms.PSet(algorithm   = cms.string('gap'),
+                                                                       TkGapClusParameters = cms.PSet(zSeparation = cms.double(0.2)  # 0.2 cm max separation betw. clusters
+                                                                                                      ) 
+                                                                       )
+                                           )
+
+####################################################################
+# Path
+####################################################################
+process.p = cms.Path(process.goodvertexSkim*
+                     process.offlineBeamSpot*
+                     #process.MeasurementTrackerEvent*
+                     process.TrackRefitter*
+                     process.PVValidation)
+
+"""
+
+####################################################################
+####################################################################
+PVValidationScriptTemplate="""
+#!/bin/bash 
+source /afs/cern.ch/cms/caf/setup.sh
+eos='/afs/cern.ch/project/eos/installation/cms/bin/eos.select'
+
+echo  -----------------------
+echo  Job started at `date`
+echo  -----------------------
+
+export theLabel=.oO[alignmentName]Oo.
+export theDate=.oO[runboundary]Oo.
+
+cwd=`pwd`
+cd .oO[CMSSW_BASE]Oo./src
+export SCRAM_ARCH=.oO[SCRAM_ARCH]Oo.
+eval `scram runtime -sh`
+cd $cwd
+
+rfmkdir -p .oO[datadir]Oo.
+rfmkdir -p .oO[workingdir]Oo.
+rfmkdir -p .oO[logdir]Oo.
+rm -f .oO[logdir]Oo./*.stdout
+rm -f .oO[logdir]Oo./*.stderr
+
+if [[ $HOSTNAME = lxplus[0-9]*\.cern\.ch ]] # check for interactive mode
+then
+    rfmkdir -p .oO[workdir]Oo.
+    rm -f .oO[workdir]Oo./*
+    cd .oO[workdir]Oo.
+else
+    mkdir -p $cwd/TkAllInOneTool
+    cd $cwd/TkAllInOneTool
+fi
+
+.oO[CommandLine]Oo.
+
+ls -lh . 
+
+$eos mkdir -p /store/caf/user/$USER/.oO[eosdir]Oo./plots/
+for RootOutputFile in $(ls *root )
+do
+    xrdcp -f ${RootOutputFile}  root://eoscms//eos/cms/store/caf/user/$USER/.oO[eosdir]Oo./
+    rfcp ${RootOutputFile}  .oO[workingdir]Oo.
+done
+
+cp .oO[CMSSW_BASE]Oo./src/Alignment/OfflineValidation/macros/FitPVResiduals.C .
+cp .oO[CMSSW_BASE]Oo./src/Alignment/OfflineValidation/macros/CMS_lumi.C .
+cp .oO[CMSSW_BASE]Oo./src/Alignment/OfflineValidation/macros/CMS_lumi.h .
```

Comment:
@mmusich I think, @hroskes made some effort to avoid the need of checking out the OfflineValidation package
Could you, at least in the next update, consider modifying this to get the files from `$CMSSW_RELEASE_BASE` in case the package is not checked out?